platform/kube/inject/inject_test.go

```diff
@@ -152,6 +152,14 @@ func TestIntoResourceFile(t *testing.T) {
 			in:   "testdata/replicationcontroller.yaml",
 			want: "testdata/replicationcontroller.yaml.injected",
 		},
+		{
+			in:   "testdata/hello-ibm.yaml",
+			want: "testdata/hello-ibm.yaml.inject",
+		},
+		{
+			in:   "testdata/hello-google.yaml",
+			want: "testdata/hello-google.yaml.inject",
+		},
```

Comment:
why do you need two ? They both seem to be excluding ibm-system namespace. Might as well just have one.

Reply:
My original thinking is that i want to test the cases where the excludeNamespaces have multiple values, but yes, have one value is good enough.